qiskit/providers/ibmq/job/ibmqjob.py

```diff
@@ -279,6 +285,173 @@ def cancel(self) -> bool:
             raise IBMQJobApiError('Unexpected error when cancelling job {}: {}'
                                   .format(self.job_id(), str(error))) from error
 
+    def change_name(self, name: str) -> bool:
```

Comment:
Another alternative was `update_name`, in case it sounds better than `change_name` one?

The same would go for `modify_tags` -> `update_tags`.